src/components/LandingPages/Guests/SliderGuests.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import "./SliderGuests.css"
+import OwlCarousel from 'react-owl-carousel';
+import 'owl.carousel/dist/assets/owl.carousel.css';
+import 'owl.carousel/dist/assets/owl.theme.default.css';
+
+const SliderGuests = () => {
+  return (
+    // <section id="testimonials">
+    //   SliderGuests
+    // </section>
+
+    <section id="testimonials" className="testimony-section">
+        <div className="overlay"></div>
+        <div className="container_slider">
+            <div className="row_guest_slider justify-content-center mb-5 pb-3">
+                <div className="col-md-7 text-center heading-section_guests heading-section-white ftco-animate" style={{marginTop: "25px"}}>
+                    <h3 className="subheading_guests" style={{color: '#6db549',fontWeight: '900'}}>What They're Saying</h3>
+                    <h2 className="mb-1 sub_slider_guests" style={{color: 'white'}}>Testimonials</h2>
+            </div>
+        </div>
+
+        <div className="row_guest_slider ftco-animate_slider">
+            <div className="col-md-12">
```

Comment:
```suggestion
            <div className="col-12">
```
Probably this should work